app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/infowindow/InfoWindowActivity.kt

```diff
@@ -0,0 +1,108 @@
+package com.mapbox.maps.testapp.examples.markersandcallouts.infowindow
+
+import android.graphics.Bitmap
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.MapView
+import com.mapbox.maps.Style
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.plugin.annotation.generated.*
+import com.mapbox.maps.plugin.gestures.OnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.addOnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.removeOnMapLongClickListener
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.utils.BitmapUtils
+import java.text.DecimalFormat
+
+/**
+ * Test activity showcasing using the legacy InfoWindow API above Washington D.C
+ * using new View Annotation API.
+ *
+ * @see [legacy example code](https://github.com/mapbox/mapbox-gl-native-android/blob/main/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/infowindow/InfoWindowActivity.java)
+ */
+class InfoWindowActivity : AppCompatActivity(), OnMapLongClickListener {
+
+  private lateinit var mapView: MapView
+  private lateinit var icon: Bitmap
+
+  private lateinit var markerManager: MarkerManager
+  private var customMarker: Marker? = null
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    mapView = MapView(this)
+    setContentView(mapView)
+
+    icon = BitmapUtils.bitmapFromDrawableRes(
+      this@InfoWindowActivity,
+      R.drawable.blue_marker_view
+    )!!
+    mapView.getMapboxMap().apply {
+      setCamera(
+        cameraOptions {
+          center(Point.fromLngLat(-77.03655168667463, 38.897705003219784))
+          zoom(15.0)
+        }
+      )
+      loadStyleUri(Style.MAPBOX_STREETS) {
```

Comment:
Replace with `getStyle`

Reply:
why?

Reply:
We would load `MAPBOX_STREETS` by default, no need to set it.